packages/update/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Handler } from 'aws-lambda';
+import { getConfig } from '@guardian/transcription-service-common';
+import { sendEmail, getSESClient } from './ses';
+import { IncomingSQSEvent } from './sqs-event-types';
+
+const messageBody = (
+	transcriptId: string,
+	transcript: string,
+	originalFilename: string,
+	rootUrl: string,
+): string => {
+	const exportUrl = `${rootUrl}/export/${transcriptId}`;
+	return `
+		<h1>Transcript for ${originalFilename} ready</h1>
+		<p>Click <a href="${exportUrl}">here</a> to export to a google doc.</p>
+		<h2>Transcript</h2>
+		${transcript
```

Comment:
Good point, with a link we give the journalist the option of whether they want the file to go into google or not rather than including in the email body.

I think maybe this is nice for making any demos we do a bit more impressive, but agreed we should remove and replace with export to google doc, and maybe also just a 'download file' link